extensions/format/defaults.lisp

```diff
@@ -0,0 +1,53 @@
+(defpackage #:lem-format/defaults
+  (:use :cl :alexandria-2 :lem :lem-format))
```

Comment:
Generally it is seen as bad practive to "use" a big library like alexandria. You can create a package-local-nickname or cherry-pick imports.